Jenkinsfile

```diff
@@ -31,27 +31,48 @@ pipeline {
         stage('TJobC IdResource: basket-api catalog-api chrome-browser eshopUser identity-api webmvc ') {
           steps {
             catchError(buildResult: 'UNSTABLE', stageResult: 'FAILURE') {
-              sh '$SCRIPTS_FOLDER/tjoblifecycles/tjob-setup.sh tjobc 0'
-              sh '$SCRIPTS_FOLDER/tjoblifecycles/tjob-testexecution.sh tjobc 0 http:// 5000 "CatalogTests#addProductsToBasket"'
-              sh '$SCRIPTS_FOLDER/tjoblifecycles/tjob-teardown.sh tjobc 0'
+              script {
+                try{
+                  sh '$SCRIPTS_FOLDER/tjoblifecycles/tjob-setup.sh tjobc 0'
+                  sh '$SCRIPTS_FOLDER/tjoblifecycles/tjob-testexecution.sh tjobc 0 http:// 5000 "CatalogTests#addProductsToBasket"'
+                  sh '$SCRIPTS_FOLDER/tjoblifecycles/tjob-teardown.sh tjobc 0'
+                } catch (err) {
+                  sh '$SCRIPTS_FOLDER/tjoblifecycles/tjob-teardown.sh tjobc 0'
+                  throw err
+                } // EndCatch
```

Comment:
The approach to manage exceptions is different than the used in retorch-st-fulteaching:
```
            catchError(buildResult: 'UNSTABLE', stageResult: 'FAILURE') {
              sh '$SCRIPTS_FOLDER/tjoblifecycles/tjob-setup.sh tjobc 0'
              sh '$SCRIPTS_FOLDER/tjoblifecycles/tjob-testexecution.sh tjobc 0 https://full-teaching- 5000 "FullTeachingEndToEndRESTTests#attendersRestOperations"'
              sh '$SCRIPTS_FOLDER/tjoblifecycles/tjob-teardown.sh tjobc 0'
            }// EndExecutionStageErrorTJobC
```

Reply:
I've made the necessary updates in the RETORCH project's base. Once this PR would be accepted I'll also ensure that the changes are reflected in https://github.com/giis-uniovi/retorch-st-fullteaching/pull/80."

Reply:
Yes, this was redundant. Initially, I believed that 'catcherror' only specified the exception behavior of the step. However, it has now been correctly implemented